lib/ecmascript.ts

```diff
@@ -735,7 +735,7 @@ export function ParseTemporalTimeZoneStringRaw(timeZoneString: string) {
   throwBadTimeZoneStringError(timeZoneString);
 }
 
-export function ParseTemporalTimeZoneString(stringIdent: string) {
+function ParseTemporalTimeZoneString(stringIdent: string) {
```

Comment:
This might still be used by validStrings.mjs, see my other comment.

Reply:
Actually, it's `ParseTemporalTimeZoneStringRaw` that's used in validStrings.mjs. 

```js
const parsingMethod = ES[`ParseTemporal${mode}StringRaw`] ?? ES[`ParseTemporal${mode}String`];
```

So AFAIK this removed export is correct.